checks/branch_protected.go

```diff
@@ -246,10 +246,11 @@ func requiresThoroughReviews(protection *github.Protection, branch string, dl ch
 		protection.RequiredPullRequestReviews.RequireCodeOwnerReviews {
 		return true
 	}
-	switch {
-	case protection.RequiredPullRequestReviews == nil:
+	if protection.RequiredPullRequestReviews == nil {
 		dl.Warn("Pullrequest reviews disabled on branch '%s'", branch)
-		fallthrough
```

Comment:
To avoid panic, we can just remove `fallthrough`.
But to separate pre-requirement check (i.e. `protection.RequiredPullRequestReviews == nil`) and scoring (other logics in `switch`), I move it on independent if statement.